Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.svc.ci.openshift.org/openshift/release:golang-1.12 AS builder
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.13 AS builder
```

Comment:
I'd drop this whole file as we have done in other repos. There is no OKD yet and registry.svc.ci.openshift.org/openshift/origin-v4.0:base is actually OCP anyways

Reply:
CI was using wrong dockerfile, pls drop it in a followup